### ISSUE TITLE
Compiling with Visual Studio 2015

### ISSUE THREAD

**Issue by bigakoz** [nmoehrle/mvs-texturing#68]
I am trying to compile with VS2015 after creating the project with CMake.

I get various compile errors as follows:

: error C3016: 'i': index variable in OpenMP 'for' statement must have signed integral type
calculate_data_costs.cpp(101): error C2672: 'multi_gauss_unnormalized': no matching overloaded function found
etc

Will it compile with VS2015 or should I try another compiler. The instructions mention GCC or compatible compiler.

Regards

**Comment by nmoehrle** [comment]
Andre maintains a visual studio compatible [fork](https://github.com/andre-schulz/mvs-texturing/network).

**Comment by bigakoz** [comment]
The Fork doesn't make much sense to me. Just goes to a Network table.

There is one note about using openMP 2.0 instead of openMP3.0+. Is that the solution.

or is it better to compile with MinGW or similar ?

**Comment by nmoehrle** [comment]
It was the network of the fork though ^^ Sorry let me try again.

Andre's cmake [branch](https://github.com/andre-schulz/mvs-texturing/tree/cmake) contains some tweaks and other win compatibility patches and should build with visual studio.

The issue is indeed visual studios lacking support for higher versions of OpenMP.

**Comment by bigakoz** [comment]
Thanks this worked perfectly.

However I am having trouble passing the correct parameters to TexRecon.

I am using the openMVG and MVE pipeline.
I call openMVG_main_openMVG2MVE2 to create the data for the MVE pipeline.
I can run all the MVE functions but I get
"No proper input scene descriptor given."

texrecon %MyPath%\mve/ %MyPath%\mve/output_mesh_clean.ply %MyPath%\mve/output_mesh_textured.ply

I substitute My actual path with %MyPath% for clarity

Any suggestions

**Comment by bigakoz** [comment]
I worked it out. I found some python code that called texrecon

texrecon %MyPath%\mve::undistorted %MyPath%\mve/output_mesh_clean.ply
